src/new/wallet.js

```diff
@@ -384,22 +384,29 @@ class HathorWallet extends EventEmitter {
    * Currently does not have support to send custom tokens, only HTR
    *
    * @param {Array} outputs Array of outputs with each element as an object with {'address', 'value'}
+   * @param {Object} token Token object {'uid', 'name', 'symbol'}. Optional parameter if user already set on the class
```

Comment:
Not ideal here that we have to send the full object, instead of just the `uuid`. But I was looking into this and it seems it's required in the end by `wallet.prepareSendTokensData`, even though we just use the `uuid` there also. The name and symbol are only used for log messages. But I guess there's no better way to do it now.

Reply:
I totally agree with you, my first idea was to expect only the token UID but the lib method expects the full object as you said. I thought about receiving the UID and then sending a request to the full node to get the full data but I thought it would be too much only because of log messages, then I left like this

Reply:
Yeah, I don't think there's much we can do now.